packages/main/src/Menu.ts

```diff
@@ -263,22 +266,68 @@ class Menu extends UI5Element {
 		return this.shadowRoot!.querySelector<ResponsivePopover>("[ui5-responsive-popover]")!;
 	}
 
+	get _list() {
+		return this.shadowRoot!.querySelector<List>("[ui5-list]");
+	}
+
+	get _menuItemGroups() {
+		return this.items.filter((item) : item is MenuItemGroup => item.isGroup);
+	}
+
 	get _menuItems() {
-		return this.items.filter((item): item is MenuItem => !item.isSeparator);
+		return this.items.filter((item) : item is MenuItem => !item.isSeparator && !item.isGroup);
+	}
+
+	get _menuItemsAll() {
+		const items: Array<MenuItem> = [];
+
+		this.items.forEach(item => {
+			if (item.isGroup) {
+				items.push(...(item as MenuItemGroup)._menuItems);
+			} else if (!item.isSeparator) {
+				items.push(item as MenuItem);
+			}
+		});
+
+		return items;
+	}
+
+	get _menuItemsNavigation() {
```

Comment:
I would suggest _navigatableMenuItems, as this name suggest that we return a "Navigation"

Reply:
Done.